package.json

```diff
@@ -32,33 +33,37 @@
     "glob": "^7.1.1"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "~2.18.2",
+    "broccoli-asset-rev": "^2.7.0",
+    "ember-cli": "~3.2.0",
     "ember-cli-autoprefixer": "^0.8.1",
     "ember-cli-dependency-checker": "^2.0.0",
     "ember-cli-eslint": "^4.2.1",
     "ember-cli-github-pages": "0.2.0",
     "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-page-object": "^1.13.0-alpha.1",
-    "ember-cli-qunit": "^4.1.1",
+    "ember-cli-page-object": "1.15.0-beta.3",
```

Comment:
using of beta is required to use page objects with `async`/`await`

Reply:
Mm.. seems like it currently works only with page object master including https://github.com/san650/ember-cli-page-object/pull/416.
Will update here once page object is published

Reply:
👍 Thanks!